.github/workflows/docker_publish.yml

```diff
@@ -75,7 +74,7 @@ jobs:
         uses: firehed/multistage-docker-build-action@v1
         with:
           repository: ${{ env.image_base_tag }}
-          stages: base, external_deps, hdf5, moab, dagmc
+          stages: base, external_deps, dagmc, final
           server-stage: dagmc_test
```

Comment:
Sorry, I'd like to ask my previous question here. I believe the final stage could be set as `server-stage`, and `dagmc_test` may be moved to `stages` part.

Reply:
Should I put `dagmc_test` to stages and `final` to server-stage?

Reply:
I don't think it makes a lot of different for our purposes.

Reply:
Then we are good to go.